src/assets/base.css

```diff
@@ -41,6 +41,7 @@
 }
 
 body {
+  padding: 40px;
```

Comment:
Please don't add padding to the body as later on the minigames should not have a 40px space between the graphics and the screen border.
Rather add to your HomeView container